buildSrc/src/main/kotlin/Versions.kt

```diff
@@ -20,7 +20,7 @@ object Versions {
     const val SHADOW = "7.1.2"
 
     // https://github.com/linkedin/dex-test-parser/releases
-    const val DEX_TEST_PARSER = "2.3.3"
+    const val DEX_TEST_PARSER = "2.3.4"
```

Comment:
This will make the following PR redundant: https://github.com/Flank/flank/pull/2223
This update with resolve the following issue: https://github.com/Flank/flank/issues/2203
FYI @bootstraponline [dmitry-ryadnenko-izettle](https://github.com/dmitry-ryadnenko-izettle)

Reply:
I merged the dep update PRs.